data/persistence/round_counts_per_year.json

```diff
@@ -0,0 +1 @@
+{"19":"409"}
```

Comment:
Actually accurate at the time of writing this.

Reply:
If Pomf is to be believed, server started in May 2013, SO WHAT HAPPENED TO SEASONS 1 TO 12, HUH?

Reply:
Pomf accidentally overwrote the VHS tapes

Reply:
Lynch Pomf when?

Reply:
Why wouldn't pomf be believed?

Reply:
Also we've had 29865 rounds since may 2013

Reply:
in the year
29865

Reply:
just for fun here's the monthly round count since i started hosting
![asdf](http://ss13.moe/uploads/2019-02-04_13-49-49.png)

Reply:
What happened in late 2014?

Reply:
November 2014 Badclown Hosting Incident